### DISCUSSION TITLE
Orchard Menu degault theme show only 1 level of menu

### DISCUSSION THREAD

**Discussion by yassinsayed** [OrchardCMS/OrchardCore#12283] [Q&A]
the default menu of OrchardCore is showing only one level of the menu where we need to have multiple levels can you please support
![OMenu level](https://user-images.githubusercontent.com/34830420/187024677-4890f04f-9ca0-4126-a3eb-62f85dabfaba.gif)

**Comment by hishamco** [comment]
Please check the Blog Theme it seems support multi-level menu
